### ISSUE TITLE
Timestamp in Log Detail lacks precision & doesn't reflect Preferences

### ISSUE THREAD

**Issue by philrz** [brimdata/brim#1642]
Repro is with Brim commit a52d915.

Importing the attached [sample.pcap.gz](https://github.com/brimdata/brim/files/6480393/sample.pcap.gz) (after uncompressing), we see that the timestamp in the Log Detail view differs significantly from the one in the main events panel. Specifically:

1. The precision (`.328`) has been dropped.
2. It shows the time relative to UTC (`-07:00`). The default **Preferences** setting for **Timezone** is **UTC**, so based on that I'd expect the format ending in `Z` like we see in the main events panel.

![image](https://user-images.githubusercontent.com/5934157/118309455-2564d980-b4a2-11eb-8bf7-5d747b708c3b.png)

The context in which this became significant is that I'm updating some Support articles that advise users on how to debug pcap problems in the Brimcap era. To that end, I was trying to manually reconstruct a `brimcap search` command line equivalent to the one the app would launch by cut & pasting the values from the Log Detail panel. I see now that if I'd done a right-click **Copy** I actually would have gotten the format I was expecting, and in a pinch I could stress in the article that the user take this approach and advise against using keyboard shortcuts like Cmd-C, which is what I was doing when I bumped into the problem. But even outside the cut & paste context, having the presentation in both spots line up seems like a good idea.

Incidentally, the difference in behavior between Cmd-C and right-click **Copy** reminded me of https://github.com/brimdata/brim/pull/1535 where it was noted that there was similar unfinished business regarding the stripping of commas when data was headed to the paste buffer. I bumped into this again when copying the port values like `id.orig_p`, though it was pretty easy to spot and remove the comma in that case. A comment was made in that PR:

> Since the data viewer is due for an upgrade, we will consider filling in the holes when that upgrade takes place.

Maybe this implies that now is still not the time to make fixes to Log Detail?

**Comment by jameskerr** [comment]
Thanks for opening @philrz. I definitely don't think there should be commas in the port field. I'll put up a fix for both the timestamp format and the commas in the ports.

**Comment by philrz** [comment]
Verified in Brim commit 4ea31e1.

As shown in the attached video, now the timestamp presentation format in Log Detail matches the one in the main events view. A cut & paste using keyboard shortcuts still grabs the value "as is", while a right-click **Copy** gets the full precision with the trailing `Z` to indicate UTC. The commas have also been removed from the port fields, and as mentioned in the linked PR, more colors for different data types have also made their way into the Log Detail view. Finally, I also verified that the presentation still matches after changing the timezone in **Preferences** to something other than UTC, and the right-click **Copy** still grabs the full-precision UTC presentation, which seems good.

https://user-images.githubusercontent.com/5934157/118541972-35c9be00-b707-11eb-82d4-4f006664bfbd.mp4

Thanks @jameskerr!
